src/headers.ts

```diff
@@ -0,0 +1,57 @@
+export const allowedRequestHeaders = [
+  'Accept',
+  'Accept-Charset',
+  'Accept-Encoding',
```

Comment:
We can't proxy those 3 headers since we forming request to real GraphQL server our self and parsing response our self. So if a client adds some of these headers I'm not sure that `node-fetch` will correctly parse response.